shell/platform/embedder/embedder.h

```diff
@@ -1549,6 +1608,33 @@ FlutterEngineResult FlutterEngineSendPointerEvent(
     const FlutterPointerEvent* events,
     size_t events_count);
 
+//------------------------------------------------------------------------------
+/// @brief      Send a key event to the engine, causing the framework to
+///             dispatch a key event that is simply transformed from this one.
```

Comment:
I would change the whole sentence to just "Sends a key event to the engine." The fact that it's internally translated and dispatched is, I think, pretty obvious.